src/services/DiscordService.ts

```diff
@@ -127,16 +127,15 @@ export class DiscordService {
 
     if (user) {
       try {
-        const profile = await profiles.load(user)
-        const hasDclProfile = !!profile && !profile.isDefaultProfile
-        const profileHasName = hasDclProfile && !!profile.name && profile.name.length > 0
+        const profile = await getProfile(user)
+        const profileHasName = !!profile && !!profile.name && profile.name.length > 0
```

Comment:
shouldn't you also check that it's not the default profile? Or use `hasDclProfile` instead

`const profileHasName = hasDclProfile && !!profile.name && profile.name.length > 0`

Which is the same we are doing in the hook, and makes me think it could be part of the return variables

Reply:
isDefaultProfile was a dcl-gatsby add-on. In this case it is not necessary!